src/inner-slider.js

```diff
@@ -118,6 +127,11 @@ export class InnerSlider extends React.Component {
       this.callbackTimers.forEach(timer => clearTimeout(timer));
       this.callbackTimers = [];
     }
+    if(this.props.touchMove) {
+      this.list.removeEventListener("touchmove", this.swipeMove, {
+        passive: false
```

Comment:
good to check if `animating` is true, if its true you can have passive true which will improve performance
as you are promising the handler won't call `preventDefault` to disable scrolling.